tests/pipelines/stable_diffusion/test_stable_diffusion_img2img.py

```diff
@@ -212,211 +213,215 @@ def test_stable_diffusion_img2img_num_images_per_prompt(self):
 
 @slow
 @require_torch_gpu
-class StableDiffusionImg2ImgPipelineIntegrationTests(unittest.TestCase):
+class StableDiffusionImg2ImgPipelineSlowTests(unittest.TestCase):
```

Comment:
Img2Img is the most flaky, the results don't match even with 3 steps. If I won't find the cause today, will raise the tolerances

Reply:
Sounds good 1e-3 is totally fine. FYI in transformers we only have 5e-3 for one-forward pass tests